src/component/NMRium.tsx

```diff
@@ -227,8 +219,8 @@ function NMRium({
       toggle(false);
     },
   });
-  const [isRightPanelHide, hideRightPanel] = useState(!initialShowPanels);
-  const [isResizeEventStart, setResizeEventStart] = useState(false);
+
+  const [isResizeEventStart] = useState(false);
```

Comment:
can we remove this state? it seems useless now if it cannot be changed

Reply:
I don't remember in which case we are hiding on load the panel but it is something we could need in the future.

Not a must have before merging this PR